src/handlers/metricsHandlers.js

```diff
@@ -20,6 +29,16 @@ export class BucketsHandler {
         const Buckets = new ListMetrics('buckets');
         return Buckets.getTypesMetrics(utapiRequest, cb);
     }
+
+    /**
+    * List metrics for the most recent 15 minutes of a given list of buckets
+    * @param {UtapiRequest} utapiRequest - UtapiRequest instance
+    * @param {callback} cb - callback
+    * @return {undefined}
+    */
+    static listRecentMetrics(utapiRequest, cb) {
```

Comment:
These handlers duplicate a lot of code. Can you see if you can use one class with a static method and parameterize the metric level (service/accounts/buckets/users)